components/dashboard/src/menu/Menu.tsx

```diff
@@ -52,64 +47,49 @@ export default function Menu() {
-        title: "Admin",
-        link: "/admin",
-        alternatives: [
-            ...getAdminTabs().reduce(
-                (prevEntry, currEntry) =>
-                    currEntry.alternatives
-                        ? [...prevEntry, ...currEntry.alternatives, currEntry.link]
-                        : [...prevEntry, currEntry.link],
-                [] as string[],
-            ),
-        ],
-    };
+    const adminMenu: Entry = useMemo(
+        () => ({
+            title: "Admin",
+            link: "/admin",
+            alternatives: [
+                ...getAdminTabs().reduce(
+                    (prevEntry, currEntry) =>
+                        currEntry.alternatives
+                            ? [...prevEntry, ...currEntry.alternatives, currEntry.link]
+                            : [...prevEntry, currEntry.link],
+                    [] as string[],
+                ),
+            ],
+        }),
+        [],
+    );
 
-    const handleFeedbackFormClick = () => {
+    const handleFeedbackFormClick = useCallback(() => {
         setFeedbackFormVisible(true);
-    };
+    }, []);
 
-    const onFeedbackFormClose = () => {
+    const onFeedbackFormClose = useCallback(() => {
         setFeedbackFormVisible(false);
-    };
+    }, []);
 
     return (
         <>
-            <header className="app-container flex flex-col pt-4 space-y-4" data-analytics='{"button_type":"menu"}'>
-                <div className="flex h-10 mb-3">
-                    <div className="flex justify-between items-center pr-3">
-                        <Link to="/" className="pr-3 w-10">
+            <header className="app-container flex flex-col pt-4" data-analytics='{"button_type":"menu"}'>
+                <div className="flex justify-between h-10 mb-3 w-full">
+                    <div className="flex items-center">
+                        {/* hidden on smaller screens */}
+                        <Link to="/" className="hidden md:inline pr-3 w-10">
                             <img src={gitpodIcon} className="h-6" alt="Gitpod's logo" />
                         </Link>
                         <OrganizationSelector />
-                        <div className="pl-2 text-base text-gray-500 dark:text-gray-400 flex max-w-lg overflow-hidden">
-                            {leftMenu.map((entry) => (
-                                <div className="p-1" key={entry.title}>
-                                    <PillMenuItem
-                                        name={entry.title}
-                                        selected={isSelected(entry, location)}
-                                        link={entry.link}
-                                    />
-                                </div>
-                            ))}
+                        {/* hidden on smaller screens (in it's own menu below on smaller screens) */}
+                        <div className="hidden md:block pl-2">
+                            <OrgPagesNav />
```

Comment:
**thought**: Looking forward for the changes in https://github.com/gitpod-io/gitpod/pull/16433 to land and tone down a bit these two links here.